Wizardry/Core/StatScreen/DrawPages/DrawPage1.c

```diff
@@ -118,115 +118,112 @@ STATIC_DECLAR void DrawStatWithBarRework(int num, int x, int y, int base, int to
 extern u8 gStatScreenStatsGrowthsToggle; 
 inline STATIC_DECLAR int DisplayGrowths(void) 
 { 
-	return gStatScreenStatsGrowthsToggle;
+    return gStatScreenStatsGrowthsToggle;
 } 
 
 STATIC_DECLAR int GetGrowthBonusOffset(int diff) 
 { 
-	diff = ABS(diff); 
-	int result = 1; 
-	if (diff > 99) 
+    diff = ABS(diff); 
+    int result = 1; 
+    if (diff > 99) 
     { 
         result++; 
     } 
-	return result; 
+    return result; 
 } 
 
 STATIC_DECLAR void DrawGrowthWithDifference(int x, int y, int base, int modified)
 {
     int diff = modified - base;
-	int offset = 0; 
-	if (base > 99) 
+    int offset = 0; 
+    if (base > 99) 
     { 
         offset++; 
     } 
-	if ((base < 100) && (diff < 100)) 
+    if ((base < 100) && (diff < 100)) 
     { 
         offset++; 
     } 
     PutNumberOrBlank(gUiTmScratchA + TILEMAP_INDEX(x + offset, y), TEXT_COLOR_SYSTEM_BLUE, base);
-	offset += GetGrowthBonusOffset(diff); 
+    offset += GetGrowthBonusOffset(diff); 
     PutNumberBonus(diff, gUiTmScratchA + TILEMAP_INDEX(x + offset, y));
 }
 
 struct TalkEventCond 
```

Comment:
There has been struct definitions in vanilla. follow the vanilla function `CheckForCharacterEvents`

Reply:
I looked at that function but I was unable to understand how the decomp parses the struct I am using here.